WMFComponents/Sources/WMFComponents/Components/Year in Review/View Models/WMFYearInReviewViewModel.swift

```diff
@@ -383,17 +383,19 @@ public class WMFYearInReviewViewModel: ObservableObject {
                                 tappedInfo: tappedInfo
                             )
                             
-                            readCountSlideV3 = WMFYearInReviewSlideStandardViewModel(
-                                gifName: "personal-slide-01",
-                                altText: localizedStrings.personalizedYouReadAccessibilityLabel,
-                                title: localizedStrings.personalizedYouReadSlideTitleV3(readData.readCount, readData.minutesRead),
-                                subtitle: localizedStrings.personalizedYouReadSlideSubtitleV3(readData.readCount),
-                                subtitleType: .html,
-                                infoURL: aboutYiRURL,
-                                forceHideDonateButton: false,
-                                loggingID: "read_count_custom",
-                                tappedInfo: tappedInfo
-                            )
+                            if readData.readCount > 255 {
```

Comment:
I think Product wants to only remove the percentage text if they have <= 255 (that is, if they would not be in the top 50% at least). But this conditional would hide the slide entirely in that case. Can you check with them?